.github/workflows/dockerfile-to-ecr.yaml

```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - main
+      - CNPT-727-proxy-rewrite-urls
```

Comment:
Please delete this line from the push to main

Reply:
good catch. Removed